app/models/registration.rb

```diff
@@ -313,49 +313,44 @@ def registration_history
   end
 
   def to_v2_json(admin: false, history: false, pii: false)
+    private_attributes = pii ? %w[dob email] : nil
```

Comment:
this won't work because the pii data is assumed to be at the base of the json and not in the user field

Reply:
To fix it you would need to include the frontend changes from this PR
https://github.com/thewca/worldcubeassociation.org/pull/10286/files

Reply:
Thanks, done! :heavy_check_mark: